website/docs/docs/deploy/hybrid-setup.md

```diff
@@ -0,0 +1,159 @@
+2. Select the **Artifact upload** button and copy the following values, which the dbt Core user will need to reference in their dbt Core's `dbt_project.yml` configuration:
+   - **[Tenant URL](/docs/cloud/about-cloud/access-regions-ip-addresses)**
+   - **Account ID**
+   - **Environment ID**
+   - **Create a service token**
+     - dbt Cloud creates a service token with **Job Runner** or **Job Viewer** permissions.
+     - Note if you don't see the **Create service token** button, it's likely you don't have the necessary permissions to create a service token. Contact your dbt Cloud admin to either get the necessary permissions or create the service token for you.
+<Lightbox src="/img/docs/deploy/hp-artifact-upload.png" title="Generate hybrid project service token" />
+
+3. Make sure to copy and save the values as they're needed to configure your dbt Core project in the next step. Once the service token is created, you can't access it again.
+
+### Configure dbt Core project and upload artifacts
+
+Once you have the values from the previous step, you can prepare your dbt Core project for artifact upload by following these steps:
+
+1. Check your dbt version by running `dbt --version` and you should see the following:
+   ```bash
+      Core:
+      - installed: 1.10.0-b1
+      - latest:    1.9.3     - Ahead of latest version!
+   ```
+2. If you don't have the latest version, [upgrade](/docs/core/pip-install#change-dbt-core-versions) your dbt Core project by running `python -m pip install --upgrade dbt-core`.
+3. Set the following environment variables in your dbt Core project by running the following commands in the CLI. Replace the `your_account_id`, `your_environment_id`, and `your_token` with the actual values in the [previous step](#generate-service-token-and-artifact-upload-values).
+
+   ```bash
+   export DBT_CLOUD_ACCOUNT_ID=your_account_id
+   export DBT_CLOUD_ENVIRONMENT_ID=your_environment_id
+   export DBT_CLOUD_TOKEN=your_token
+   export DBT_UPLOAD_TO_ARTIFACTS_INGEST_API=True
+   ```
+
+   - Set the environment variables in whatever way you use them in your project.
+   - To unset an environment variable, run `unset environment_variable_name`, replacing `environment_variable_name` with the actual name of the environment variable.
+
+4. In your local dbt Core project, add the following items you copied in the [previous section](/docs/deploy/hybrid-setup#enable-artifact-upload) to the dbt Core's `dbt_project.yml` file:
+   - `tenant_hostname`
+   - `DBT_CLOUD_ACCOUNT_ID` &mdash; uses a `DBT_CLOUD_` prefix
+   - `DBT_CLOUD_ENVIRONMENT_ID` &mdash; uses a `DBT_CLOUD_` prefix
+   ```yaml
+   name: "jaffle_shop"
+   version: "3.0.0"
+   require-dbt-version: ">=1.5.0"
+   ....rest of dbt_project.yml configuration...
+
+   dbt-cloud:
+   tenant_hostname: cloud.getdbt.com # Replace with your Tenant URL
```

Comment:
DBT_CLOUD_ACCOUNT_ID and DBT_CLOUD_ENVIRONMENT_ID should not be specified in the project.yml file, they will be obtained via env vars.

Reply:
thanks @pratikmallya ! so should the project.yml file look like the below only:

   ```yaml
   name: "jaffle_shop"
   version: "3.0.0"
   require-dbt-version: ">=1.5.0"
   ....rest of dbt_project.yml configuration...

   dbt-cloud:
   tenant_hostname: cloud.getdbt.com # Replace with your Tenant URL
   ```

Reply:
yes. It should be indented though:
```
dbt-cloud:
  tenant_hostname: cloud.getdbt.com # Replace with your Tenant URL
```